rdr_service/dao/study_nph_dao.py

```diff
@@ -175,10 +175,15 @@ def get_stored_samples_subquery(
                 return stored_samples_subquery.filter(Participant.id == nph_participant_id)
 
             if query_def := kwargs.get('query_def'):
-                if query_def.field_filters:
+                if applicable_filters := \
+                    [
+                        filter_obj for filter_obj
+                        in query_def.field_filters
+                        if filter_obj.field_name in ['modified']
+                     ]:
```

Comment:
What do you think of removing the extra line break to simplify it a bit, and to help the indenting of the last bracket a bit?

```suggestion
                if applicable_filters := [
                    filter_obj for filter_obj
                    in query_def.field_filters
                    if filter_obj.field_name in ['modified']
                ]:
```

Reply:
Sounds good!